test/lib/bamboo/plug/sent_email_viewer_plug_test.exs

```diff
@@ -190,6 +190,44 @@ defmodule Bamboo.SentEmailViewerPlugTest do
     assert conn.resp_body =~ "Email not found"
   end
 
+  test "list emails over API" do
+    normalize_and_push_pair(:html_email,
```

Comment:
Thoughts on using `normalize_and_push` to create a single email since the assertions are all about the first one? Or alternatively doing assertions on both of the emails?